platform/mbed_lib.json

```diff
@@ -90,6 +90,12 @@
             "value": null
         },
 
+        "stack-dump-enabled": {
+            "macro_name": "MBED_STACK_DUMP_ENABLED",
+            "help": "Set to 1 or true to enable stack dump.",
+            "value": null
```

Comment:
I know you're following the bad pattern of the things above, but options like this should be `true` or `false`. `null` is undefined. Let's get this new one right, and fix the others later. (They've come up an issue already - I think they actually use `ifdef` so interpret `false` as "on"?)

Reply:
Got it. I changed with https://github.com/ARMmbed/mbed-os/pull/11332/commits/1e601e55adcea670b82b98c8fd5364211926a84b.